lib/assets/javascripts/views/devise/registrations/edit.js

```diff
@@ -44,4 +44,9 @@ $(() => {
   $('#password-confirmation #user_current_password').change((e) => {
     $('#password-confirmation #user_password').val($(e.target).val());
   });
+
+  // Submit the form when the user clicks the confirmation button on the modal
+  $('#password-confirmation #pwd-confirmation').click((e) => {
```

Comment:
could you please remove the first part of the query selector (e.g. #password-confirmation)? you don't need to look for children of password-confirmation since id MUST be unique across the page. If we have duplicate ids across a page, this should be changed otherwise HTML validators would complain.

Reply:
sure. we had some issues in the past with some of the inputs auto-generated by the Rails helpers (e.g. #user_id) and JS for pages like edit profile affecting the login modal for example. They're not necessary here though so will remove.

Reply:
Can change that one, but the others need to be explicit like that otherwise they'll affect the login/create account modals (e.g. `$('#password-confirmation #user_current_password').change()`